docs/overview.md

```diff
@@ -4,6 +4,9 @@ Monokle is laid out like many other desktop applications:
 
 ![Monokle UI](img/monokle-ui-1.4.0.png)
 
+(On start-up, Monokle automatically loads the most recently selected folder.)
```

Comment:
... On start-up, Monokle automatically loads the most recently selected folder if the corresponding "Load recent folder on Startup" setting has been enabled ?

Reply:
Changed.